### PR TITLE
Fix syntax error

### PR DESCRIPTION
**Pull request by jblueh**
## Overview

Fixes a syntax error in #18

## Related work

#18

### CODE REVIEW

**Review by pcarruscag — APPROVED** [review]

**Comment by pcarruscag** [comment]
@jblueh I've made you a maintainer of this Repo, if you need to merge PR's related to tsan you can go ahead without approval (I think maintainer should be enough for that, if not let me know).

**Comment by jblueh** [comment]
Thanks @pcarruscag :+1:

**Pull request merged**
